pkg/operator/controller/garden/garden/reconciler_reconcile.go

```diff
@@ -532,6 +532,10 @@ func (r *Reconciler) reconcile(
 
 					patch := client.MergeFrom(garden.DeepCopy())
 					garden.Status.EncryptedResources = shared.StringifyGroupResources(encryptedResources)
```

Comment:
Add a TODO when this can be removed.
```suggestion
                    // TODO ...
					garden.Status.EncryptedResources = shared.StringifyGroupResources(encryptedResources)
```